### PR TITLE
Finish the \a escape sequence

### PR DESCRIPTION
**Pull request by r00ster91**

### CODE REVIEW

File: spec/compiler/lexer/lexer_spec.cr [hunk 1]
```diff
@@ -231,8 +231,8 @@ describe "Lexer" do
   it_lexes_number :i8, ["0i8", "0"]
 
   it_lexes_char "'a'", 'a'
-  it_lexes_char "'\\a'", 7.chr
-  it_lexes_char "'\\b'", 8.chr
+  it_lexes_char "'\\a'", '\u{7}'
```
Comment (straight-shoota) [inline spec/compiler/lexer/lexer_spec.cr]: Why not `'\a'`?

Comment (r00ster91) [reply spec/compiler/lexer/lexer_spec.cr]: I did this because `p 7.chr` gives `'\u{7}'`

Comment (bcardiff) [reply spec/compiler/lexer/lexer_spec.cr]: @r00ster91 can you change the `\u{7}` to `\a` and also use `\b` in the following spec to keep it consistent with the rest of the suite.

**Comment by bcardiff** [comment]
For reference, this is a follow up of #5864

**Comment by r00ster91** [comment]
Okay changed it.

**Comment by straight-shoota** [comment]
You should also add `\a` and `\b` to `Char#dump_or_inspect`.

**Comment by r00ster91** [comment]
That Travis CI failed is just a runtime error I think

**Pull request merged**